src/benchmark_environments/testing/envs.py

```diff
@@ -184,3 +184,40 @@ def test_premature_step(env: gym.Env, skip_fn, raises_fn) -> None:
     act = env.action_space.sample()
     with raises_fn(Exception):  # need to call env.reset() first
         env.step(act)
+
+
+class CountingEnv(gym.Env):  # pragma: no cover
```

Comment:
Why the `pragma: no cover`?

Reply:
Left over from copying, doesn't seem to be useful here so I'll get rid of it